torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -606,11 +1020,15 @@ void PrepareInplaceOpsForONNX(const std::shared_ptr<Graph>& graph) {
   PrepareListAppendAndInsertForONNX(graph->block());
```

Comment:
how does the new pass `RegisterInplaceOpAsBlockOutputs` work with `PrepareInplaceOpsForONNX`?

Reply:
Prepare inplace ops is now doing preparations for index_put, list, etc. We need to merge these two parts gradually.